src/components/Dropdown/Dropdown.tsx

```diff
@@ -25,6 +25,7 @@ export const dropdownClassNames = {
   button: `${baseClassName}_button`,
   popover: `${baseClassName}_popover`,
   item: `${baseClassName}_item`,
+  actionMenuItem: `${baseClassName}_actionMenuItem`,
```

Comment:
I'd rather call this hideActiveSelection etc. so that it's not too strongly connected to action menu use case. The functionality itself looks good!

Reply:
Good point. Changed the name.